src/features/applicationView/offerCard/OfferCard.tsx

```diff
@@ -171,54 +176,57 @@ const OfferCard = ({
                 <LabelValuePair label={t('offer.berthDetails.comment')} value={berthComment} />
               </Section>
             </Section>
-            <Section title={t('offer.billing.title').toUpperCase()}>
-              {order && (
-                <>
-                  <Section>
-                    <LabelValuePair
-                      label={t('offer.billing.basePrice')}
-                      value={formatPrice(order.price, i18n.language)}
-                    />
-                    {order.fixedProducts.map((product, i) => (
-                      <LabelValuePair
-                        key={i}
-                        label={t(getProductServiceTKey(product.name))}
-                        value={formatPrice(product.price, i18n.language)}
-                      />
-                    ))}
-                  </Section>
-                  <Section>
+            {order && (
+              <Section title={`${t('offer.billing.title').toUpperCase()} nro: ${order.orderNumber}`}>
+                <Section>
+                  <LabelValuePair
+                    label={t('offer.billing.basePrice')}
+                    value={formatPrice(order.price, i18n.language)}
+                  />
+                  {order.fixedProducts.map((product, i) => (
                     <LabelValuePair
-                      label={t('offer.billing.additionalServices')}
-                      value={
-                        <button onClick={() => setIsEditing(true)}>
-                          <Text color="brand">{t('common.edit')}</Text>
-                        </button>
-                      }
+                      key={i}
+                      label={t(getProductServiceTKey(product.name))}
+                      value={formatPrice(product.price, i18n.language)}
                     />
-                    {order.optionalProducts.map((product, i) => (
-                      <LabelValuePair
-                        key={i}
-                        label={t(getProductServiceTKey(product.name))}
-                        value={formatPrice(product.price, i18n.language)}
-                      />
-                    ))}
-                  </Section>
-                  <hr />
-                  <Section>
+                  ))}
+                </Section>
+                <Section>
+                  <LabelValuePair
+                    label={t('offer.billing.additionalServices')}
+                    value={
+                      <button onClick={() => setIsEditing(true)}>
+                        <Text color="brand">{t('common.edit')}</Text>
+                      </button>
+                    }
+                  />
+                  {order.optionalProducts.map((product, i) => (
                     <LabelValuePair
-                      label={t('offer.billing.total').toUpperCase()}
-                      value={formatPrice(order.totalPrice, i18n.language)}
+                      key={i}
```

Comment:
I would suggest using procut.id as a key if it is available. Using the index is not recommended

Reply:
Done